PORTAL/jobs/__main__.py

```diff
@@ -845,7 +847,6 @@ def _add_queue_cli(add_cmd: Callable, add_hidden: bool = True) -> Callable:
     sub = add_cmd('queue', help='Manage the job queue')
     queue = sub.add_subparsers(
         dest='queue_cmd',
-        metavar='CMD',
```

Comment:
Doesn't removing this add noise to the help output?

Reply:
I suppose it's kind of subjective.  I saw the `CMD` as sort of a confusing extra word and I didn't know what it meant.

Before this change:

```
usage: benchmark [-h] [-v] [-q] CMD ...

options:
  -h, --help     show this help message and exit
  -v, --verbose
  -q, --quiet

subcommands:
  CMD
    list         Print a table of known jobs
    show         Print a summary of the given (or current) job
    run-bench    Create a benchmarking request and run it
    attach       Tail a job's log file
    cancel       Stop the current job (or prevent a pending one)
    upload       Upload benchmark results to the public data store
    compare      Compare two or more results

Full documentation is available at
https://github.com/microsoft/faster-cpython/wiki/Benchmarking-Tooling
```

After this change:

```
usage: benchmark [-h] [-v] [-q] {list,show,run-bench,attach,cancel,upload,compare} ...

options:
  -h, --help            show this help message and exit
  -v, --verbose
  -q, --quiet

subcommands:
  {list,show,run-bench,attach,cancel,upload,compare}
    list                Print a table of known jobs
    show                Print a summary of the given (or current) job
    run-bench           Create a benchmarking request and run it
    attach              Tail a job's log file
    cancel              Stop the current job (or prevent a pending one)
    upload              Upload benchmark results to the public data store
    compare             Compare two or more results

Full documentation is available at
https://github.com/microsoft/faster-cpython/wiki/Benchmarking-Tooling
```

Reply:
I like the second better, since the unnecessary line is at least understandable. A line saying just `CMD` without any explanation is just mysterious.